playbooks/secrets-init.yml

```diff
@@ -47,7 +47,7 @@
           when: vault_init|default('') == 'plugin'
 
         - name: Populate secrets.yml with missing secrets
-          ansible.builtin.shell: "{{ base_folder }}/scripts/generate-secret.sh -s {{ item }} -m {{ gs_mode }} >> {{ secrets_file }}"
+          ansible.builtin.shell: "bash {{ base_folder }}/scripts/generate-secret.sh -s {{ item }} -m {{ gs_mode }} >> {{ secrets_file }}"
```

Comment:
This has not proven to be any useful. Please revert

Reply:
It does not impact any changes so same work with or without bash

Reply:
please be conservative and only comit changes we know had value of fix things